docs/platform/howto/byoc/assign-project-custom-cloud.rst

```diff
@@ -19,34 +32,45 @@ Prerequisites
 * At least one :doc:`custom cloud created </docs/platform/howto/byoc/create-custom-cloud>` in your Aiven organization
 * Access to `Aiven Console <https://console.aiven.io/>`_
 
-Assign projects
----------------
+Enable projects to use your custom cloud
+----------------------------------------
 
 1. Log in to `Aiven Console <https://console.aiven.io/>`_ as an administrator.
-2. From the left sidebar, select **Bring your own cloud**.
-3. In the **Bring you own cloud** view, select one of the clouds available on the list.
-4. In the selected cloud's page, navigate to the **Projects availability** tab and select **Assign projects**.
-5. In the **Assign projects** window, use the dropdown menu to select a project you want to assign to your cloud.
-6. Confirm your choice by selecting **Assign projects**.
+2. Select the organization you want to use from the dropdown menu in the top right corner.
+3. From the top navigation bar, select **Admin**.
+4. From the left sidebar, select **Bring your own cloud**.
+5. In the **Bring your own cloud** view, select one of the clouds available on the list.
+6. In the selected cloud's page, navigate to the **Available projects** tab and modify the settings provided as needed:
+   
+   * Select **Set availability** to decide if your custom cloud is available in all the projects in your organization or in selected projects only. In the **Custom cloud's availability in your organization** window, select either **By default for all projects** or **By selection**. If you go for the **By selection** option, dropdown menus **Assign organizational units** and **Assign projects** show up. Use them to select desired organizational units and/ or projects and confirm your choice by selecting **Save**.
+
+   .. note::
+
+      By selecting an organization unit, you make your custom cloud available from all the projects in this unit.
```

Comment:
In at least some other places we have "organizational unit" instead of "organization unit", I don't think there's too high of a chance of a confusion, but consistency would be nice regardless.